src/atopile/front_end.py

```diff
@@ -75,10 +76,12 @@ def __repr__(self) -> str:
         return f"<{self.__class__.__name__} {self.address}>"
 
 
+from attrs import define, field
+
 @define
 class Physical(Base):
     """Let's get physical!"""
-    unit: pint.Unit
+    unit: pint.Unit = field(converter=lambda u: str(u) if isinstance(u, pint.Unit) else u)
```

Comment:
This is typed as a `Pint` unit. It should be converted to a string like this on ingestion. Whatever is using this should do the conversion to a string - or, if this does warrant changing, the typing needs changing too